doc/concepts/index.rst

```diff
@@ -21,25 +21,41 @@ The python front-end further abstracts away some of these steps for single cell
 describe the cell and simulation; and the details of the recipe and computational resources construction are
 handled under the hood. Generally speaking though, these 3 steps are the building blocks of an Arbor application.
 
-:ref:`Recipes <modelrecipe>` represent a set of neuron constructions and connections with *mechanisms* specifying ion channel and synapse dynamics in a cell-oriented manner. This has the advantage that cell data can be initiated in parallel.
+:ref:`Recipes <modelrecipe>` represent a set of neuron constructions and connections with *mechanisms* specifying
+ion channel and synapse dynamics in a cell-oriented manner. This has the advantage that cell data can be initiated
+in parallel.
 
-A cell represents the smallest unit of computation and forms the smallest unit of work distributed across processes. Arbor has built-in support for different :ref:`cell types <modelcells>`, which can be extended by adding new cell types to the C++ cell group interface.
+A cell represents the smallest unit of computation and forms the smallest unit of work distributed across processes.
+Arbor has built-in support for different :ref:`cell types <modelcells>`, which can be extended by adding new cell
+types to the C++ cell group interface.
 
-:ref:`modelsimulation` manage the instantiation of the model and the scheduling of spike exchange as well as the integration for each cell group. A cell group represents a collection of cells of the same type computed together on the GPU or CPU. The partitioning into cell groups is provided by :ref:`modeldomdec` which describes the distribution of the model over the locally available computational resources.
+:ref:`modelsimulation` manage the instantiation of the model and the scheduling of spike exchange as well as the
+integration for each cell group. A cell group represents a collection of cells of the same type computed together
+on the GPU or CPU. The partitioning into cell groups is provided by :ref:`modeldomdec` which describes the distribution
+of the model over the locally available computational resources.
 
-In order to visualize the result of detected spikes a spike recorder can be used and to analyse Arbor's performance a meter manager is available.
+In order to visualize the result of detected spikes a spike recorder can be used and to analyse Arbor's performance a
+meter manager is available.
+
+Arbor concepts
+==============
 
 .. toctree::
    :maxdepth: 2
-   :caption: Arbor concepts:
+   :caption: From recipe to simulation:
 
    recipe
    cell
-   cable_cell
-   morphology
-   labels
-   mechanisms
    interconnectivity
    hardware
    domdec
    simulation
+
```

Comment:
Have you tested this breakup of the TOC? Won't that create two separate concept sections, 'From recipe to simulation:' and 'The cable cell:'?

Reply:
That'll create 2 labeled sections on the main `Arbor concepts` page, but wont change anything on the toolbar on the lhs of the screen.

Reply:
The index on the lhs is created from the TOCs in `doc/*/index.rst`. See the TOC in `doc/index.rst`. I think there may be a hide directive, but you should test this because it gets a bit finicky.

Reply:
I have tested it. And the index on the lhs is still the same.